products/bash/release-ace.sh

```diff
@@ -16,6 +16,7 @@
 #   -n : <namespace> (string), Defaults to "cp4i"
 #   -r : <dashboard-release-name> (string), Defaults to "ace-dashboard-demo"
 #   -e : <designer-release-name> (string), Defaults to "ace-designer-demo"
+#   -c : <releaseBoth> (string), Default to releasing both ace dashboard and designer
```

Comment:
This options sounds very confusing. So I think if not specified it releases both dashboard and designer, and if specified it releases only dashboard? That sounds very confusing. We don't want to add confusing options and then have to support them from now on

Reply:
Split the release ace into two scripts and called both in release ace and release dashboard from the prereqs for the tekton pipelines.